### PR TITLE
Add xda-developers.com

### PR DESCRIPTION
**Pull request by adinklotz**
SimilarWeb rank is 2986
Unfortunately couldn't find an actual documentation page to link, but the option is there
Screenshot 2023-10-03 at 5 08 40 PM

### CODE REVIEW

File: img/x/xda-developers.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" id="Layer_1" x="0" y="0" version="1.1" viewBox="0 0 26.9 33.5"><style id="style1" type="text/css">.st2{fill:#bcbcbc}</style><g id="g3" transform="translate(0 -.1)"><g id="g1"><path id="path1" d="M16 4.4V.1h10.9v24.5H16v-4.2h6.4v-16z" style="fill:#2daae9"/></g><path id="polygon1" d="M0 .1v24.5h5.5l5.4 9V20.4H4.5v-16h6.4V.1z" style="fill:#e43f64"/></g></svg>
```
Comment (Carlgo11) [inline img/x/xda-developers.com.svg:1]: Here's a version that won't cause the test warnings.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.9 33.5"><path fill="#2daae9" d="M16 4.3V0h10.9v24.5H16v-4.2h6.4v-16z"/><path fill="#e43f64" d="M0 0v24.5h5.5l5.4 9V20.3H4.5v-16h6.4V0z"/></svg>
```

File: entries/x/xda-developers.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Site Name": {
```
Comment (Carlgo11) [inline entries/x/xda-developers.com.json:2]: Please change "Site Name" to the actual name of the website/service.

File: entries/x/xda-developers.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "domain": "xda-developers.com",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "categories": [
+      "social"
```
Comment (Carlgo11) [inline entries/x/xda-developers.com.json:9]: What type of social service does the website provide?
Please note that we do not list forums so if that's the only service that requires authentication, then the site can't be listed.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/x/xda-developers.com.json (hunk 3 above)
Comment (adinklotz) [reply entries/x/xda-developers.com.json:9]: Oh I guess I missed that requirement. Yeah it's a forum, so we can close this :(

**Pull request closed without merging**